Recrutify/Recrutify.Host/Controllers/CandidatesController.cs

```diff
@@ -97,5 +97,14 @@ public async Task<ActionResult> BulkCreateTestFeedbacksAsync(BulkCreateTestFeedb
             await _candidateService.BulkCreateTestFeedbacksAsync(bulkCreateTestFeedbackDTO);
             return NoContent();
         }
+
+        //[Authorize(Policy = Constants.Policies.FeedbackPolicy)]
```

Comment:
create new policy ManagerPolicy and use it here

Reply:
done